docker_scripts/squash.py

```diff
@@ -518,8 +520,8 @@ def run(self):
             # without loading into Docker
             self._tar_image(self.output_path, new_image_dir)
             self.log.info("Image available at '%s'" % self.output_path)
-        else:
-            # Load image into Docker
+        if not self.output_path or self.load_output_back:
```

Comment:
Ok, so just to check - if user provides `load_output_back=False, self.output_path=False`, nothing happens in that case? Or should `Squash.__init__` raise some kind of error if both of these are False?